src/main/java/org/jenkinsci/plugins/ghprb/extensions/comments/GhprbPublishJenkinsUrl.java

```diff
@@ -33,13 +33,9 @@ public String getPublishedURL() {
     }
 
     public String postBuildComment(AbstractBuild<?, ?> build, TaskListener listener) {
-        StringBuilder msg = new StringBuilder();
-
-        msg.append("\nRefer to this link for build results (access rights to CI server needed): \n");
-        msg.append(generateCustomizedMessage(build));
-        msg.append("\n");
-        
-        return msg.toString();
+        return "\nRefer to this link for build results (access rights to CI server needed): \n" +
```

Comment:
The StringBuilder is cleaner and more readable.  It also allows for easier expansion or editing.